src/poparser.js

```diff
@@ -77,8 +77,8 @@ Parser.prototype._handleCharset = function (buf = '') {
   let match;
 
   if ((pos = str.search(/^\s*msgid/im)) >= 0) {
-    pos = pos + str.substr(pos + 5).search(/^\s*(msgid|msgctxt)/im);
-    headers = str.substr(0, pos >= 0 ? pos + 5 : str.length);
+    pos = pos + str.substring(pos + 5).search(/^\s*(msgid|msgctxt)/im);
+    headers = str.substring(0, pos >= 0 ? pos + 5 : str.length);
```

Comment:
those changes aren't required, but will throw an error if not changed due the typescript language level set to ES2015

Reply:
Also good catch! Also pre-TS PR please.